drivers/video/Kconfig.arducam_mega

```diff
@@ -0,0 +1,12 @@
+# ARDUCAM_MEGA
+
+# Copyright (c) 2020 FrankLi Limited
+# SPDX-License-Identifier: Apache-2.0
+
+config VIDEO_ARDUCAM_MEGA
+	bool "Arducam Mega CMOS digital image sensor"
+	select SPI
+	depends on DT_HAS_ARDUCAM_MEGA_ENABLED
+	default y
```

Comment:
default y ?

Reply:
@marcusfolkesson 
Thank you for your detail check.
Yes, Here we refer to this [video drivers](https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/video/). we find all of those driver's configuration are defaut y.